packages/terra-application-name/src/ApplicationHeaderName.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+
+import styles from './ApplicationHeaderName.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The accessory element to be displayed next to the title.
+   * */
+  accessory: PropTypes.element,
+  /**
+   * The title branding of the application name.
+   * */
+  title: PropTypes.string,
+};
+
+const ApplicationHeaderName = ({
+    accessory,
+    title,
+    ...customProps
+  }) => {
+  const ApplicationHeaderNameClassNames = cx([
+    'application-header-name',
+    customProps.className,
+  ]);
+
+  return (
+    <div {...customProps} className={ApplicationHeaderNameClassNames}>
+      {!!accessory && <div className={cx('accessory')}>{accessory}</div>}
```

Comment:
Do we need the double !! for this? In other code I've written, I've been able to get my desired functionality with just `{accessory && <div>{accessory}</div>}`